sdk/resourcemanager/aad/armaad/CHANGELOG.md

```diff
@@ -1,239 +1,9 @@
 # Release History
 
 ## 1.0.0 (2022-05-17)
-### Breaking Changes
 
-- Function `*PrivateEndpointConnectionsClient.BeginCreate` return value(s) have been changed from `(*armruntime.Poller[PrivateEndpointConnectionsClientCreateResponse], error)` to `(*runtime.Poller[PrivateEndpointConnectionsClientCreateResponse], error)`
-- Function `*PrivateEndpointConnectionsClient.BeginDelete` return value(s) have been changed from `(*armruntime.Poller[PrivateEndpointConnectionsClientDeleteResponse], error)` to `(*runtime.Poller[PrivateEndpointConnectionsClientDeleteResponse], error)`
-- Function `*PrivateLinkForAzureAdClient.BeginCreate` return value(s) have been changed from `(*armruntime.Poller[PrivateLinkForAzureAdClientCreateResponse], error)` to `(*runtime.Poller[PrivateLinkForAzureAdClientCreateResponse], error)`
-- Function `ErrorDefinition.MarshalJSON` has been removed
-- Function `PrivateLinkResourceProperties.MarshalJSON` has been removed
-- Function `PrivateLinkPolicyListResult.MarshalJSON` has been removed
-- Function `PrivateEndpointConnectionListResult.MarshalJSON` has been removed
-- Function `PrivateLinkResourceListResult.MarshalJSON` has been removed
+The package of `github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/aad/armaad` is using our next generation design principles since version 1.0.0, which contains breaking changes.
```

Comment:
next generation design principles link to https://azure.github.io/azure-sdk/general_introduction.html